protocol/packages/swap/src/trx/mod.rs

```diff
@@ -3,8 +3,10 @@ use finance::coin::{Amount, CoinDTO};
 use platform::{ica::HostAccount, trx::Transaction};
 use sdk::cosmos_sdk_proto::cosmos::base::abci::v1beta1::MsgData;
 
-#[cfg(feature = "testing")]
-pub use self::test::*;
```

Comment:
it's important the tests to see the request and response types not in `test` module

Reply:
Why not in there, when we explicitly show them on `testing`? If we do that, then `test` module should go or be private and only contain tests.